pkgs/os-specific/linux/kernel/common-config.nix

```
@@ -29,7 +29,7 @@ let
     mkIf (stdenv.hostPlatform.isAarch32 ||
           stdenv.hostPlatform.isAarch64 ||
           stdenv.hostPlatform.isx86_64 ||
-          (stdenv.hostPlatform.isPowerPC && stdenv.hostPlatform.is64bit) ||
+          (stdenv.hostPlatform.isPower32BigEndian && stdenv.hostPlatform.is64bit) ||
```

I guess this should rather read:

```suggestion
          (stdenv.hostPlatform.isPower && stdenv.hostPlatform.is64bit) ||
```

---

Sure.  If #168111 merges we can shorten this even further.